scheduler/system_sched.go

```diff
@@ -276,11 +276,6 @@ func (s *SystemScheduler) computePlacements(place []allocTuple) error {
 		node, ok := nodeByID[missing.Alloc.NodeID]
 		if !ok {
 			s.logger.Debug("could not find node %q", missing.Alloc.NodeID)
-			if s.failedTGAllocs == nil {
```

Comment:
just to call out, this was introduced by https://github.com/hashicorp/nomad/pull/6968 and is not the desired functionality or any functionality that exists in a released version of nomad